src/index.html

```diff
@@ -6,7 +6,69 @@
     <title>Calendar</title>
     <link rel="stylesheet" href="./styles/main.scss">
   </head>
-  <body>
-    <h1>Calendar</h1>
+  <body class="page">
+    <div class="calendar">
```

Comment:
Check the task description, you should add modifiers for the calendar for managing start day and month length

Reply:
Set some modifiers as the initial ones